pkg/scheduler/framework/events.go

```diff
@@ -101,6 +101,22 @@ var (
 	CSIStorageCapacityAdd = ClusterEvent{Resource: CSIStorageCapacity, ActionType: Add, Label: "CSIStorageCapacityAdd"}
 	// CSIStorageCapacityUpdate is the event when a CSI storage capacity is updated in the cluster.
 	CSIStorageCapacityUpdate = ClusterEvent{Resource: CSIStorageCapacity, ActionType: Update, Label: "CSIStorageCapacityUpdate"}
+	// PodSchedulingContextAdd is the event when a pod scheduling context is added.
+	PodSchedulingContextAdd = ClusterEvent{Resource: PodSchedulingContext, ActionType: Add, Label: "PodSchedulingContextAdd"}
```

Comment:
Should `framework/plugins/dynamicresources/dynamicresources.go` use this variable?

Reply:
AFAIK no. I don't see any plugin using this predefined variables (in plugins the logical OR is used widely when defining the `ActionType`, so it would be hard to use these).